tasks/eth/014-fjord-gas-config/VALIDATION.md

```diff
@@ -0,0 +1,103 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of setting the recommended protocol version and the two ownership transfers from the Foundation Operations to the Upgrade Safe (TODO).
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0x229047fed2591dbec1eF1118d64F7aF3dB9EB290` (`SystemConfig`)
+
+We overrid the ownership of the SC contract (task#011).
```

Comment:
Would prefer to use SystemConfig instead of SC to avoid confusion with the Security Council